src/main/java/com/redhat/devtools/lsp4ij/features/codeAction/LSPLazyCodeActionIntentionAction.java

```diff
@@ -62,13 +62,13 @@ public void setLazyCodeActions(LSPLazyCodeActionProvider lazyCodeActions) {
     @Override
     public @IntentionName @NotNull String getText() {
         loadCodeActionIfNeeded();
-        return title;
+        return title != null ? title : "";

```

Comment:
These are annotated as `@NotNull` but I was seeing runtime errors where they could be null. I've made them defensive, but perhaps these intentions should not ever be created if they don't have sufficient info to be used?